### DISCUSSION TITLE
Unable to update Repository Secret

### DISCUSSION THREAD

**Discussion by dinesh-thiyagarajan** [community/community#131832] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi folks, I have a Repository secret which I wanted to update. but if I update it and open it again all im seeing is an empty value , tried from both safari and chrome same issue. anyone else facing this ?

**Comment by davevad93** [comment, marked as answer]
Hi @dinesh-thiyagarajan ,

That's the normal behavior, when you click `Edit` to change the value of a repo secret you can't see its value, you can only update it.

**Reply by davevad93** [reply]
If you really need to see a secret value you could do it within a GitHub Action by running the following command in your workflow:

```yml
echo ${{secrets.YOUR_SECRET}} | sed 's/./& /g'
```

**Reply by dinesh-thiyagarajan** [reply]
Sure thanks for your help @davevad93
